public/static/js/subscriptions.js

```diff
@@ -179,17 +183,17 @@ function subCtrl($scope, $mdDialog, mciUserSettingsService) {
         return false;
     };
 
-
     $scope.bindTrigger = function() {
       _.each($scope.c.triggers, function(trigger){
         if (trigger.trigger === $scope.trigger.trigger) {
```

Comment:
I messed this up previously, but this should also check if the types match too. This is causing some nonsensical options such as task regexes for builds

Reply:
The trigger type (task, build, version). This is only checking the trigger name, but you can have the "outcome" trigger for a task, build, or version

Reply:
Done